hub/policy/src/main/java/uk/gov/ida/hub/policy/domain/controller/AuthnFailedErrorStateController.java

```diff
@@ -1,12 +1,7 @@
 package uk.gov.ida.hub.policy.domain.controller;
 
 import com.google.common.base.Optional;
-import uk.gov.ida.hub.policy.domain.AuthnRequestSignInProcess;
-import uk.gov.ida.hub.policy.domain.FailureResponseDetails;
-import uk.gov.ida.hub.policy.domain.ResponseFromHub;
-import uk.gov.ida.hub.policy.domain.ResponseFromHubFactory;
-import uk.gov.ida.hub.policy.domain.StateController;
-import uk.gov.ida.hub.policy.domain.StateTransitionAction;
+import uk.gov.ida.hub.policy.domain.*;
```

Comment:
As a tiny style point - the convention here is to prefer explicit imports to wildcard imports, so we should probably undo this.

(I think it makes sense too, especially as many of our libraries have clashing namespaces)

Reply:
Some help for how to set it up here - https://gds-way.cloudapps.digital/manuals/programming-languages/java.html#imports

Reply:
It's IntelliJ's default behaviour to "optimise" imports by doing this - and I've not changed that. Thanks @willp-bl for the link.